EIPS/eip-3333.md

```diff
@@ -0,0 +1,51 @@
+---
+eip: 3333
+title: Increase block rewards to 3 ETH, with 2 Year Decay to 1 ETH Scheduled
```

Comment:
```suggestion
title: Set block reward to 3 ETH with a 2 Year Decay to 1 ETH
```
Try to phrase things throughout the EIP in the context of a future reader who may not have a knowledge of historical state, and also so they can be implemented without needing to know previous state (e.g., this could be implemented on a chain that currently has a 5 ETH block reward or a chain with a 1 ETH block reward).

Reply:
Copy, good change